jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesTest.java

```diff
@@ -776,351 +706,346 @@ public void testGetChildValue_nullValue() {
+    Truth8.assertThat(mavenProjectProperties.getJarArtifact())
+        .isEqualTo(Paths.get("/foo/bar/helloworld-1.jar"));
   }
 
   @Test
   public void testGetJarArtifact_originalJarCopiedIfSpringBoot() throws IOException {
     temporaryFolder.newFile("helloworld-1.jar.original");
-    Mockito.when(mockBuild.getDirectory()).thenReturn(temporaryFolder.getRoot().toString());
-    Mockito.when(mockBuild.getFinalName()).thenReturn("helloworld-1");
+    when(mockBuild.getDirectory()).thenReturn(temporaryFolder.getRoot().toString());
+    when(mockBuild.getFinalName()).thenReturn("helloworld-1");
 
     setUpSpringBootFatJar();
     Path tempDirectory = temporaryFolder.newFolder("tmp").toPath();
-    Mockito.when(mockTempDirectoryProvider.newDirectory()).thenReturn(tempDirectory);
+    when(mockTempDirectoryProvider.newDirectory()).thenReturn(tempDirectory);
 
-    Assert.assertEquals(
-        tempDirectory.resolve("helloworld-1.original.jar"),
-        mavenProjectProperties.getJarArtifact());
+    Truth8.assertThat(mavenProjectProperties.getJarArtifact())
+        .isEqualTo(tempDirectory.resolve("helloworld-1.original.jar"));
 
     mavenProjectProperties.waitForLoggingThread();
-    Mockito.verify(mockLog)
-        .info("Spring Boot repackaging (fat JAR) detected; using the original JAR");
+    verify(mockLog).info("Spring Boot repackaging (fat JAR) detected; using the original JAR");
   }
 
   @Test
   public void testGetJarArtifact_originalJarIfSpringBoot_differentDirectories() throws IOException {
-    Mockito.when(mockMavenProject.getBasedir()).thenReturn(new File("/should/ignore"));
-    Mockito.when(mockBuild.getDirectory()).thenReturn("/should/ignore");
-    Mockito.when(mockBuild.getFinalName()).thenReturn("helloworld-1");
+    when(mockMavenProject.getBasedir()).thenReturn(new File("/should/ignore"));
+    when(mockBuild.getDirectory()).thenReturn("/should/ignore");
+    when(mockBuild.getFinalName()).thenReturn("helloworld-1");
 
-    Mockito.when(mockMavenProject.getPlugin("org.apache.maven.plugins:maven-jar-plugin"))
+    when(mockMavenProject.getPlugin("org.apache.maven.plugins:maven-jar-plugin"))
         .thenReturn(mockPlugin);
-    Mockito.when(mockPlugin.getExecutions()).thenReturn(Arrays.asList(mockPluginExecution));
-    Mockito.when(mockPluginExecution.getId()).thenReturn("default-jar");
-    Mockito.when(mockPluginExecution.getConfiguration()).thenReturn(pluginConfiguration);
+    when(mockPlugin.getExecutions()).thenReturn(Arrays.asList(mockPluginExecution));
+    when(mockPluginExecution.getId()).thenReturn("default-jar");
+    when(mockPluginExecution.getConfiguration()).thenReturn(pluginConfiguration);
     addXpp3DomChild(pluginConfiguration, "outputDirectory", Paths.get("/jar/out").toString());
 
     setUpSpringBootFatJar();
 
-    Assert.assertEquals(
-        Paths.get("/jar/out/helloworld-1.jar"), mavenProjectProperties.getJarArtifact());
+    Truth8.assertThat(mavenProjectProperties.getJarArtifact())
```

Comment:
```suggestion
   assertThat(mavenProjectProperties.getJarArtifact())
```

Reply:
Same below.

Reply:
This was intentional. I get a compilation error when using `Truth.assertThat` with `Path` due to some type ambiguity (which I think is very unusual for a library like this). Anything you know about this when asserting `Path`?